plugins/TouchIcon/class.touchicon.plugin.php

```diff
@@ -45,7 +45,7 @@ public function OnDisable() {
    public function Structure() {
       // Backwards compatibility with v1.
       if (C('Plugins.TouchIcon.Uploaded')) {
-         SaveToConfig('Garden.TouchIcon', PATH_ROOT.'/uploads/TouchIcon/apple-touch-icon.png');
+         SaveToConfig('Garden.TouchIcon', 'TouchIcon/apple-touch-icon.png');
```

Comment:
No, the 1.0 version always saved the user-uploaded icon as this name in the uploads folder. The default is a constant.

The `Plugins.TouchIcon.Uploaded` config is no longer used; we're using it here to detect preexisting installs.